src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Timestamp.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+#if IS_DESKTOP
+using System.Security.Cryptography.Pkcs;
+#endif
+
+namespace NuGet.Packaging.Signing
+{
+    public sealed class Timestamp
+    {
+#if IS_DESKTOP
+
+        /// <summary>
+        /// Upper limit of Timestamp.
+        /// </summary>
+        public DateTimeOffset UpperLimit { get; }
+
+        /// <summary>
+        /// Lower limit of Timestamp.
+        /// </summary>
+        public DateTimeOffset LowerLimit { get; }
+
+        /// <summary>
+        /// Time timestamp was created by the Time Stamp Authority.
+        /// </summary>
+        public DateTimeOffset GeneralizedTime { get; }
```

Comment:
This is never set in the constructor

Reply:
thanks, missed that in haste, just fixed.